README.md

```diff
@@ -26,17 +26,74 @@ Hologram requires the following things to already be setup:
 * Developers using Hologram must be running OS X or Linux machines. The built packages support Debian derivatives. No Windows support is planned, but patches are welcome.
 
 ## Installation
-Hologram currently doesn't ship pre-compiled binaries, so you'll need to build it yourself. This build process has only been tested on OS X Mavericks and OS X Yosemite. Because it compiles binaries for OS X, you probably can't do the building on OS X. It does, however, cross-compile to Linux just fine using gox.
-
-### Building Packages
-Hologram comes with support for packaging for Debian-based servers. To build these, do the following:
-
-1. `mkdir -p $GOPATH/src/github.com/AdRoll`
-2. `git clone git@github.com:AdRoll/hologram.git $GOPATH/src/github.com/AdRoll/hologram`
-3. `cd $GOPATH/src/github.com/AdRoll/hologram`
-4. `make setup`: This will setup dependencies for building Hologram on your workstation.
-5. Modify the `config/{agent,server}.json` files included for your particular deployment. If you edit these, they will be included in the compiled packages. You may distribute the files in any other way you may wish, but note that they must be in this format, at `/etc/hologram/agent.json` and `/etc/hologram/server.json` respectively.
-6. `make package`: This will build the Hologram programs for OS X and Linux, and build installers for each. You will need to `sudo` during this process.
+Hologram currently doesn't ship pre-compiled binaries, so you'll need to build it yourself. A docker container is provided that contains all that's needed to test, compile and build hologram packages for both debian and osx. You just need to invoke the script from the same directory where the hologram source lives. This is a full example of and testing and building packages for all supported platforms.
```

Comment:
Shouldn't that be `This is a full example of testing and building packages for all supported platforms.' :)